website/content/en/blog/2024-06-idle-power-matters-kepler-metrics-for-public-cloud-energy-efficiency.md

```diff
@@ -0,0 +1,160 @@
+- Size(vm) - size of the VM
+- P(dyn) - dynamic power range of the device
+- P(base) - Idle, or base, power consumption is when the VMM is operating but no VMs are executed on the system
+- n - the number of VMs running on the server
+
+To achieve green software building, software practitioners must collectively reduce the above power consumption and the SCI scores during the design, development, and delivery of software applications. This requires a collaborative effort to prioritize sustainability and minimize carbon emissions.
+
+## Idle Power: The Hidden Energy Cost of Software in Public Cloud Computing
+
+In 2019, [RightScale's survey](https://resources.flexera.com/web/media/documents/rightscale-2019-state-of-the-cloud-report-from-flexera.pdf) revealed that over 94% of 800 organizations had adopted cloud computing, with 31% prioritizing public cloud and 54% planning to migrate their workloads to the public cloud within 12 months to become more resource efficient. [Accenture's report in 2020](https://newsroom.accenture.com/news/2020/cloud-migrations-can-reduce-co2-emissions-by-nearly-60-million-tons-a-year-according-to-new-research-from-accenture) highlighted that public cloud migration can lead to a substantial reduction in carbon emissions (up to 84%) and energy consumption (up to 64%).
+
+
+The public cloud operates as a multi-tenant environment where users purchase computational resources shared among numerous clients. Various tools, such as [CodeCarbon](https://codecarbon.io), [CarbonTracker](https://carbontracker.org), [Scaphandre](https://github.com/hubblo-org/scaphandre), and [Kepler](https://sustainable-computing.io), are available for estimating energy consumption and carbon emissions associated with code execution wrapping or real-time resource utilization observation.
+
+
+However, determining how to attribute power consumption for infrastructure services presents a challenge. The GSF's SCI standard specifies that energy consumption should encompass all energy consumed by reserved or provisioned hardware, not solely the hardware utilized for application software needs. This includes idle power, encompassing not only static power but also the energy consumed by control plane processes such as kubelet or container runtime (e.g., containerd) in kubernetes orchestration contexts.
+
+
+According to the power data reported by [SPECpower benchmark](https://www.spec.org/power_ssj2008/), active idle power consumption can vary widely, ranging from 20% to 60% of the power consumed at maximum utilization. In absolute terms, this ranges from 100Watt to 1300Watt, equivalent to the power consumption of a classic light bulb or an air conditioner in a household (although there are differences in carbon emissions). The idle power can be even higher when considering GPU power consumption. Therefore, idle power is significant and must be allocated to each application. Then, the challenge lies in determining how to do so effectively.
```

Comment:
```suggestion
According to the [SPECpower benchmark](https://www.spec.org/power_ssj2008/), active idle power consumption can vary widely, ranging from 20% to 60% of the power consumed at maximum utilization. In absolute terms, this ranges from 100Watts to 1300Watts, equivalent to the power consumption of a classic light bulb or an air conditioner in a household (although there are differences in carbon emissions). The idle power can be even higher when considering GPU power consumption.
```

Removed a couple of sentences-not sure if they add to the paragraph significantly.